src/test/kotlin/coverosR3z/authentication/AuthenticationBDD.kt

```diff
@@ -30,20 +27,43 @@ class AuthenticationBDD {
     @Test
     fun `I can add a time entry`() {
         // given I am logged in
+        val pmd = PureMemoryDatabase()
+        val cua = CurrentUserAccessor() // We need a real cua for this test, sharing a pmd with auth persistence
```

Comment:
Probably would be good to refactor this to its own method, and also this is a lot to read through, might benefit from comments and further refactoring.

Reply:
The deed is done